apps/minifront/src/components/root-router.tsx

```diff
@@ -1,21 +1,23 @@
 import { createHashRouter, redirect } from 'react-router-dom';
 import { PagePath } from './metadata/paths';
 import { Layout } from './layout';
-import AssetsTable, { AssetsLoader } from './dashboard/assets-table';
+import AssetsTable from './dashboard/assets-table';
 import TransactionTable from './dashboard/transaction-table';
 import { DashboardLayout } from './dashboard/layout';
-import { TxDetails, TxDetailsErrorBoundary, TxDetailsLoader } from './tx-details';
+import { TxDetails, TxDetailsErrorBoundary } from './tx-details';
 import { SendLayout } from './send/layout';
-import { SendAssetBalanceLoader, SendForm } from './send/send-form';
+import { SendForm } from './send/send-form';
 import { Receive } from './send/receive';
 import { ErrorBoundary } from './shared/error-boundary';
 import { SwapLayout } from './swap/layout';
 import { SwapLoader } from './swap/swap-loader';
-import { StakingLayout, StakingLoader } from './staking/layout';
+import { StakingLayout } from './staking/layout';
 import { IbcLoader } from './ibc/ibc-loader';
 import { IbcLayout } from './ibc/layout';
+import { abortLoader } from '../abort-loader';
+import type { Router } from '@remix-run/router';
 
-export const rootRouter = createHashRouter([
+export const rootRouter: Router = createHashRouter([
```

Comment:
can we add some comments somewhere on the high-level similarities and differences between react-router loaders and zquery?

From what I can tell, both **support**:
- async data fetching
- state access to access fetched data in components

In terms of their **differences**, this is where some more clarification would be helpful. Zquery anchors on the idea of global state management, where state can be updated and shared across an application, right? React-router on the other hand fetches data specifically for a _single_ route, and rather than being stored globally, it's only accessible by a specific component. I guess I'm wondering where component-level data fetching would generally be more preferred?